chemreg/substance/tests/factories.py

```diff
@@ -0,0 +1,16 @@
+import factory
+
+from chemreg.common.factory import DjangoSerializerFactory
+from chemreg.substance.serializers import SynonymTypeSerializer
+
+
+class SynonymTypeFactory(DjangoSerializerFactory):
+    """Manufactures `SynonymType` models."""
+
+    name = factory.Faker("slug")
+    label = factory.LazyAttribute(lambda o: o.name.replace("-", " "))
+    short_description = factory.Faker("text")
+    long_description = factory.Faker("text")
+
```

Comment:
Please add the `validate_regular_expression` and `score_modifier` fields to the factory's output.